src/loaders/gtdb/gtdb_taxa_freq_loader.py

```diff
@@ -105,8 +116,15 @@ def main():
 
     print('start parsing input files')
     nodes = _parse_files(load_files)
-    docs = _create_docs(nodes, load_version, kbase_collection)
-    _convert_to_json(docs, args.output)
+    freq_docs, rank_doc = _create_docs(nodes, load_version, kbase_collection)
+    # Create taxa frequency json file
+    freq_json = args.output
+    _convert_to_json(freq_docs, freq_json)
+
+    # Create identical ranks json file
+    root_ext = os.path.splitext(freq_json.name)
+    rank_json = open(root_ext[0] + '_rank' + root_ext[1], 'w')
+    _convert_to_json(rank_doc, rank_json)
```

Comment:
I'm confused here - for the frequency json you just pass the filename, but here you pass an open file. Why the difference?

If opening a new file is necessary, better to use a `with` context manager

Reply:
Never mind, `freq_json` is an open file, misread. The `with` context manager should be added though

Reply:
👍